package.json

```diff
@@ -49,16 +49,5 @@
     "eslint-plugin-promise": "3.5.0",
     "eslint-plugin-react": "7.2.0",
     "eslint-plugin-security": "1.4.0"
-  },
-  "dependencies": {
-    "eslint-config-airbnb": "15.1.0",
-    "eslint-plugin-import": "2.7.0",
-    "eslint-plugin-jsx-a11y": "5.1.1",
-    "eslint-plugin-lodash": "2.4.4",
-    "eslint-plugin-mocha": "4.11.0",
-    "eslint-plugin-node": "5.1.1",
-    "eslint-plugin-promise": "3.5.0",
-    "eslint-plugin-react": "7.2.0",
-    "eslint-plugin-security": "1.4.0"
```

Comment:
shouldn't these be all dev deps now?

Reply:
You're right!, these should be in dev deps.